### ISSUE TITLE
Missing framework .Net

### ISSUE THREAD

**Issue by denisjoshua** [mRemoteNG/mRemoteNG#2229]
## Expected Behavior
I have download Nightly v1.77.2-nb and when I start it ask me to instal .Net
I click YES and I install .Net, then restart the PC. 

After restart I rin again the program and it give me same error: 
...you must install missing frameworks for .Net...

## Possible Solution
Witch and how I must install the .Net framework 

## Steps to Reproduce (for bugs)
1. Start program 

## Your Environment
* Version used: Nightly v1.77.2-nb
* Windows 10 x64

Thanks in advance

**Comment by Kvarkas** [comment]
hello, we are using .net 6, from [there](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) you need download [.NET Desktop Runtime 6.0.5](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-6.0.5-windows-x64-installer)

**Comment by denisjoshua** [comment]
Oh yeah ... greate support :-) 
Now it's work !!!

Maybe when I click "Yes" on the warning, in order to download the .Net 
will be better that the link to be this one that you give me. 

Thanks a lot for the help

**Comment by Kvarkas** [comment]
you are welcome ;)

**Comment by simonai1254** [comment]
On a vanilla windows (Windows 10 Sandbox) the error message is pretty clear, though I had to look really hard to get the correct version (Desktop and not only core), having installed .Net Core it's a little harder to find out what's needed.

Is there a way to bundle the installation, e.g. "we found you don't have .Net installed, should we take care of it?"?
At least one might document the requirement a little better (e.g. note in Setup assistant to install .Net as well and remark on download page of mRemoteNG)...
